### ISSUE TITLE
Метод Removable field Translatable

### ISSUE THREAD

**Issue by SWEET1S** [visual-ideas/moonshine-spatie-translatable#3]
Метод removable не работает должным образом для полей Translatable

**Comment by OzodbekUSTECH** [comment]
Yeah, same, method removable is not working for Translatable fields. Firstly, you press delete, then 'save' and everything gets back! How to solve it??
